Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/MerchantServiceConfigExtra.hs

```diff
@@ -71,7 +71,8 @@ upsertMerchantServiceConfig merchantServiceConfig = do
         [Se.Set BeamMSC.configJSON configJSON, Se.Set BeamMSC.updatedAt now]
         [ Se.And
             [ Se.Is BeamMSC.merchantId $ Se.Eq $ getId merchantServiceConfig.merchantId,
-              Se.Is BeamMSC.merchantOperatingCityId $ Se.Eq $ getId merchantServiceConfig.merchantOperatingCityId
+              Se.Is BeamMSC.merchantOperatingCityId $ Se.Eq $ getId merchantServiceConfig.merchantOperatingCityId,
+              Se.Is BeamMSC.serviceName $ Se.Eq _serviceName
```

Comment:
it appears while testing that dashboard apis change all merchants service configs for each ServiceName, that's why this condition added